### DISCUSSION TITLE
How to manipulate TaskbarIcon's properties?

### DISCUSSION THREAD

**Discussion by myfix16** [HavenDV/H.NotifyIcon#63] [Q&A] [answered]
Hi,
accessing some of TaskbarIcon's properties, such as Icon, GeneratedIcon, Margin, etc. results in a `System.Runtime.InteropServices.COMException`. So, I wonder what's the correct way to access and manipulate these values.

**Comment by HavenDV** [comment, marked as answer]
First, make sure you are using the latest version with NuGet. If yes, then this is definitely a bug, please create an issue with a detailed description of the exceptions and, if possible, with the place where they occur
